upup/models/cloudup/resources/addons/networking.weave/v1.9.0.yaml

```diff
@@ -41,19 +41,29 @@ spec:
             - name: weavedb
               mountPath: /weavedb
             - name: cni-bin
-              mountPath: /opt
+              mountPath: /host/opt
```

Comment:
Do we need any notes on the upgrade?  Also, does this path need to exist on the host?